sdk/keyvault/keyvault-keys/README.md

```diff
@@ -148,6 +148,29 @@ const url = `https://${vaultName}.vault.azure.net`;
 const client = new KeyClient(url, credential);
 ```
 
+## Specifying the Azure Key Vault service API version
+
+Our Key Vault Keys client currently only supports the Azure Key Vault service versions `7.0` and `7.1-preview`, which we specify as part of the options that can be passed through the constructor. The latest supported API version is `7.1-preview`, which is available as the `LATEST_API_VERSION` export. A use case example follows:
```

Comment:
> The latest supported API version is `7.1-preview`, which is available as the `LATEST_API_VERSION` export. 

The versioning scheme used between 7.0 and 7.1-preview makes it clear as to which is the latest one.

The `LATEST_API_VERSION` is useful to the user if they ever wanted to know which is the latest api version. They can console.log it or inspect it.
But, since the default is always the latest version, there is no reason for the user to actually use this in production.

I would recommend removing this part in favor of brevity

```suggestion
By default, this package uses the latest Azure Key Vault service version which is `7.1-preview`. The only other version that is supported is `7.0`. You can change the service version being used by setting the option `apiVersion` in the client constructor as shown below:
```

When we have more than 2 versions being supported, this can be easily modified to "The other versions that are supported are..."

Reply:
Sounds good! Thank you!